tableauserverclient/server/request_options.py

```diff
@@ -117,51 +156,47 @@ class Direction:
         Desc = "desc"
         Asc = "asc"
 
-    def __init__(self, pagenumber=1, pagesize=None):
-        self.pagenumber = pagenumber
-        self.pagesize = pagesize or config.PAGE_SIZE
-        self.sort = set()
-        self.filter = set()
 
-        # This is private until we expand all of our parsers to handle the extra fields
-        self._all_fields = False
+"""
+These options can be used by methods that are fetching data exported from a specific content item
+"""
 
-    def page_size(self, page_size):
-        self.pagesize = page_size
-        return self
 
-    def page_number(self, page_number):
-        self.pagenumber = page_number
-        return self
+class _DataExportOptions(RequestOptionsBase):
```

Comment:
nit: is `_VizExportOptions` a better class name here as I believe options like `maxAge` are limited to exporting API endpoints